### PR TITLE
Search page styling

### PR DESCRIPTION
**Pull request by mbrown20**
Making the search page look prettier

### CODE REVIEW

**Review by aniehuser — COMMENTED** [review]
Is this pull request different than the other one you submitted?

File: yw_website/website/templates/extendable/_base_workflow_listview.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{% extends 'extendable/__base_template.html' %}
```
Comment (clundin55) [inline yw_website/website/templates/extendable/_base_workflow_listview.html]: Delete this

Comment (clundin55) [inline yw_website/website/templates/extendable/_base_workflow_listview.html:1]: I don't think this file should be here

File: yw_website/yw_website/apps/website/static/design/detailed_design.css [hunk 2]
```diff
@@ -1,9 +1,7 @@
```
Comment (clundin55) [inline yw_website/yw_website/apps/website/static/design/detailed_design.css:1]: I think this styling should just be restarted off of master

File: yw_website/yw_website/apps/website/static/design/home_design.css [hunk 3]
```diff
@@ -3,18 +3,21 @@
 .list {
 	text-align: center;
 	align-items:center;
-    padding: 20px 40px 10px 0px;
+    padding: 0 40px 10px 0px;
```
Comment (clundin55) [inline yw_website/yw_website/apps/website/static/design/home_design.css:6]: should use percentages instead

File: yw_website/yw_website/apps/website/static/design/home_design.css [hunk 4]
```diff
@@ -3,18 +3,21 @@
...
     background-color: white;
-    margin:5% 10%;
+    margin: 0 10%;
```
Comment (clundin55) [inline yw_website/yw_website/apps/website/static/design/home_design.css:8]: readmore code is not working

File: yw_website/yw_website/apps/website/static/design/home_design.css [hunk 5]
```diff
@@ -81,29 +100,59 @@
...
+}
+
+.tags-label {
+	font-size: 12px;
+	margin: 10px 0 0 0;
+}
+.tags {
+	border: 1px solid rgb(5, 97, 92);;
+	display: inline-block;
+	padding: 0 5px;
```
Comment (clundin55) [inline yw_website/yw_website/apps/website/static/design/home_design.css:117]: clickable tags is cool, should try to inline tags and put them at the top. Once again you should just go off the css on master

File: yw_website/yw_website/apps/website/static/design/search.css [hunk 6]
```diff
@@ -0,0 +1,15 @@
+.search {
+    margin: 0;
+    font-family: Verdana;
+    text-align: justify;
+    border-spacing: 0 1em;
+}
```
Comment (clundin55) [inline yw_website/yw_website/apps/website/static/design/search.css:6]: This file is good. We should stop applying styling to tr and ul's it messes with other pages.

File: yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html [hunk 7]
```diff
@@ -15,45 +15,53 @@
...
-    </ol>
+
 
     <div class = outside_data>
-        <b>Description: </b>{{workflow.description}}
-        <br>
-       <b>Uploaded At: </b>{{version.last_modified}}    
+        <div class= "uploaded-at">
+            <b>Uploaded At: </b>{{version.last_modified}}
+        </div>
```
Comment (clundin55) [inline yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html:37]: this file is very far behind

File: yw_website/yw_website/apps/website/templates/includeable/workflow_list.html [hunk 8]
```diff
@@ -26,37 +25,47 @@
...
+                    </div>
+
             </li>
             <script>
                 function description_view(){
                     var description = document.getElementById("description");
-                    if (description.style.display === "none") {
-                        description.style.display = "block";
+                    var button = document.getElementById("description-button")
+                    if (window.getComputedStyle(description, null).getPropertyValue("height") === '60px') {
```
Comment (clundin55) [inline yw_website/yw_website/apps/website/templates/includeable/workflow_list.html:56]: This doesn't work, and I'm not too sure it's even worth adding

Comment (clundin55) [reply yw_website/yw_website/apps/website/templates/includeable/workflow_list.html:56]: Also the same javascript is applied to each element

File: yw_website/yw_website/apps/website/views.py [hunk 9]
```diff
@@ -45,10 +45,11 @@ def home(request):
             workflow.version_id = latest_version.id
             workflow.version_modified = latest_version.last_modified
             workflow.tags = (
-                Tag.objects.all()
-                .select_related("workflow")
-                .filter(pk=workflow.id)
```
Comment (clundin55) [inline yw_website/yw_website/apps/website/views.py:50]: This shouldn't be merged

**Review by clundin55 — CHANGES_REQUESTED** [review]
Might be better to start fresh from master, these commits are very out of date.

File: yw_website/yw_website/apps/website/templates/includeable/workflow_list.html (hunk 8 above)
Comment (aniehuser) [reply yw_website/yw_website/apps/website/templates/includeable/workflow_list.html:56]: >I'm not too sure it's even worth adding

I think it is worth adding

**Pull request closed without merging**